src/main/java/org/example/homework3/Task1.java

```diff
@@ -0,0 +1,27 @@
+package org.example.homework3;
+
+/**
+ * Program to Perform Basic Arithmetic Operations
+ */
+public class Task1 {
+    public static void main(String[] args) {
+        int num1 = 10;
+        int num2 = 5;
+
+        // Sum
```

Comment:
there is no sense in leaving a comment, like this. it does not describe the code, it is just a duplication of the variable name, please remove